src/css/blocks/post.css

```diff
@@ -1,56 +1,151 @@
 .putainde-Post {
+  color: #444;
   text-align: center;
-  padding: 0 var(--r-lineHeight);
+  padding: 0 1rem;
 }
 
-.putainde-Title {
-  z-index: 3;
+  /* if a custom header is here, we adjust the post to be near the nav */
+  .putainde-Post--customHeader {
+    margin: calc(var(--r-lineHeight) * -1) 0 0;
+  }
 
-  margin: calc(var(--r-lineHeight) / 2) 0;
-  overflow: auto;
+  .putainde-Post a {
+    color: var(--putainde-Color-red);
+    fill: var(--putainde-Color-red);
+    text-decoration: none;
+  }
+    .putainde-Post a:hover,
+    .putainde-Post a:focus {
+      text-decoration: underline;
+    }
 
-  color: #222;
-  background: #fff;
-  background: linear-gradient(to bottom, #fff 60%, color(#fff alpha(0)));
-}
+  .putainde-Post-header {
+    position: relative;
+    text-align: center;
+    margin: 0 -1rem calc(var(--r-lineHeight) * 1);
+    padding: calc(var(--r-lineHeight) * 2) 0;
 
-.putainde-Title--home {
-  text-align: center;
-}
+    /* for putainde-Post-header--dark.putainde-Post-header--filter */
+    overflow: hidden;
+  }
 
-.putainde-Title-text {
-  font-weight: 400;
-}
+    .putainde-Post-header--custom {
+      padding: 15vh 0;
+      box-shadow:
+        0 0 1rem color(#000 a(.2)) inset,
+        0 .4rem 1rem color(#000 a(.2)) inset
+      ;
+    }
 
-.putainde-Post-metas  {
-  display: inline-block;
-  font-size: var(--r-small-fontSize);
-  color: #999;
-}
+    .putainde-Post-header--dark {
+      color: #fff;
+    }
+
+    .putainde-Post-header--light {
+
+    }
+
+    .putainde-Post-header .putainde-Title {
+      padding: 0 1rem;
+    }
+
+    .putainde-Title {
+      margin: calc(var(--r-lineHeight) / 2) 0;
+      color: #222;
+      overflow: auto;
```

Comment:
I probably use this overflow for some reason I should document. Let me double check.